st2common/bin/bootstraprmq.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python2.7
+
+import sys
+import st2common.transport.bootstrap as transport_bootstrap
+
+if __name__ == '__main__':
```

Comment:
> Well, otherwise, in a distributed system we would need a coordinator
We definitely need something of this sort at some point.

> performance penalty / increase startup time
That does not bother me too much either.

> potentially calling it multiple times, is imo a better and cleaner approach.
Clean is too generic a term. Can you explain why it is a cleaner approach? If you said each process was independent then to some extent I would buy the argument. However, I do find it a complete abstraction leak when st2action is ensuring indexes for reactor objects. 

We can both make points in either direction and eventually it ends up at preference. I am overwhelmingly against doing work which need not be performed along a specific code-path. This really leads to confusing behavior in terms of questions like -
1. What the heck happened why did this get setup?
2. Whose responsibiilty is it really to perform this task? Everyones is not really a good answer.